pkg/kubectl/cmd/config/set.go

```diff
@@ -40,13 +40,28 @@ type setOptions struct {
 	setRawBytes   flag.Tristate
 }
 
-var setLong = templates.LongDesc(`
+var (
+	setLong = templates.LongDesc(`
 	Sets an individual value in a kubeconfig file
 
 	PROPERTY_NAME is a dot delimited name where each token represents either an attribute name or a map key.  Map keys may not contain dots.
 
 	PROPERTY_VALUE is the new value you wish to set. Binary fields such as 'certificate-authority-data' expect a base64 encoded string unless the --set-raw-bytes flag is used.`)
 
+	setExample = templates.Examples(`
```

Comment:
I'd prefer the examples be more cluster-type independent. Not everyone are using minikube so for those users these examples won't be of much help.

Reply:
@soltysh updated